kiva/quartz/CTFont.pyx

```diff
@@ -21,10 +21,10 @@ cdef object _cf_string_to_pystring(CFStringRef cf_string):
     if c_string == NULL:
         success = CFStringGetCString(cf_string, buf, 256,
                     kCFStringEncodingMacRoman)
-        retval = str(buf)
+        retval = bytes(buf)
     else:
-        retval = str(c_string)
-    return retval
+        retval = bytes(c_string)
+    return retval.decode('mac-roman')
```

Comment:
This is only being used for font names returned by the system. Should we perhaps look into using UTF-8 instead? I know some of the non-Latin fonts have non-Latin glyphs in their names.

Reply:
Possibly: I think this primarily for introspection of the system, rather than actively in font selection, but I've tried replacing with UTF-8 since it seems straightforward.